go.mod

```diff
@@ -34,6 +33,7 @@ require (
 	github.com/Azure/go-autorest/autorest/date v0.3.0 // indirect
 	github.com/Azure/go-autorest/logger v0.2.1 // indirect
 	github.com/Azure/go-autorest/tracing v0.6.0 // indirect
+	github.com/BurntSushi/toml v1.0.0 // indirect
```

Comment:
The only change I see above to _direct_ requirements is removal of one; why does that cause _addition_ of an indirect requirement?

Reply:
I have no clue why this was added, by go mod tidy insist on adding it. I can remove by hand, but prefer to understand why it was added

Reply:
It seems it's not related to my change:
# github.com/BurntSushi/toml
github.com/kubestellar/kubestellar/test/e2e/framework
github.com/kcp-dev/kcp/test/e2e/framework
github.com/kcp-dev/kcp/pkg/embeddedetcd
go.etcd.io/etcd/server/v3/embed
gopkg.in/natefinch/lumberjack.v2
gopkg.in/natefinch/lumberjack.v2.test
github.com/BurntSushi/toml

Reply:
This PR is adding the line here.
It might be best to commit #1014 first and then rebase.  #1014 leaves `go mod tidy` happy and has no toml in the MCCM go.mod.

Reply:
I would rather see the MCCM go.mod cleaned up than get additional junk.

Reply:
Fixed. github.com/BurntSushi/toml  removed